packages/web3-core/package.json

```diff
@@ -25,6 +25,7 @@
 		"test:integration": "jest --config=./test/integration/jest.config.js --passWithNoTests"
 	},
 	"dependencies": {
+		"web3-errors": "1.0.0-alpha.0",
```

Comment:
looks like should be 4x version

Reply:
There wasn't any errors package released in past so it will start from 1.0.0